kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -585,6 +585,28 @@ object KyuubiConf {
       .stringConf
       .createOptional
 
+  val ENGINE_TRINO_MAIN_RESOURCE: OptionalConfigEntry[String] =
+    buildConf("session.engine.trino.main.resource")
```

Comment:
seems that we have hardcoded in the shell script

Reply:
Passed through the env variable (`TRINO_ENGINE_JAR`)  of the ProcessBuilder.

Reply:
I see